cli/command/telemetry_docker.go

```diff
@@ -49,8 +51,15 @@ func dockerExporterOTLPEndpoint(cli Cli) (endpoint string, secure bool) {
 	}
 
 	// keys from https://opentelemetry.io/docs/concepts/sdk-configuration/otlp-exporter-configuration/
-	endpoint, ok = otelMap["OTEL_EXPORTER_OTLP_ENDPOINT"].(string)
-	if !ok {
+	endpoint, _ = otelMap[otelExporterOTLPEndpoint].(string)
+
+	// Override with env var value if it exists AND IS SET
+	// (ignore otel defaults for this override when the key exists but is empty)
```

Comment:
@laurazard 
I opted to not override the value here if the env var key exists but has no value, to avoid confusion with otel defaulting it's own values

Reply:
sounds good to me!

Reply:
I'd prefer if we use a different environment variable for this because of the OTEL one.

My original goal with this was to do two things:

1. Natively support the OTEL environment variables and implement it so telemetry both goes to this endpoint and to a user defined endpoint.
2. Add a custom environment variable to override the docker configuration.

Buildx currently implements 1 but I didn't want to overly complicate the original PR with that logic.

If we can prefix the environment variable with something like `DOCKER` or `DOCKER_CLI` or more preferably `DOCKER_CLI_DEBUG` I'd like that a lot better for the environment variable name.

The original environment variable has a specific usage in OTEL that's different than how it's used here and I don't want us to accidentally clobber each other.

Reply:
Thanks for the explanation @jsternberg!

My only comment here is a) should we add that logic for 1.? and b) I'd rather not have `DEBUG` in the env var given that we don't have any other `DEBUG` env vars (other than the actual `DEBUG` env var which sets log level for the CLI, so unless there's a very strong reason not to I'd rather call this something like `DOCKER_CLI_OTEL_EXPORTER_OTLP_ENDPOINT`, esp. since this server to **override** the default `OTEL_EXPORTER_OTLP_ENDPOINT` config.

Reply:
No hard feelings with this one, either of the choices is ok imho. Removed `DEBUG` from the name for now

Reply:
`DOCKER_CLI_OTEL_EXPORTER_OTLP_ENDPOINT` is good for me. As long as it doesn't conflict with the official environment variable and makes sense I'm good.